src/utils/fetchData.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	EstimatedPackage,
+	PackageOwnership,
+	tideliftMeUp,
+} from "tidelift-me-up";
+
+export type OptionsType = Record<string, unknown>;
+export type SearchParamsType = Record<string, unknown>;
```

Comment:
[Naming] Ending with `*Type` isn't common practice anymore. I'd just call them the stuff before that:

```suggestion
export type Options = Record<string, unknown>;
export type SearchParams = Record<string, unknown>;
```

...or if you want to disambiguate, maybe `DataOptions` or `FetchOptions` or `TideliftMeUpSiteOptions` or ... something. Up to you. 😄